apps/site/pages/docs/component-customization/adding-extra-properties-to-a-native-component.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: ' Adding extra properties to a native component'
```

Comment:
```suggestion
title: ' Adding extra props to a native component'
```
In the frontend development ecosystem, we just call them props, so for better SEO, I think it's better to call them what people search for.